### PR TITLE
Initial version of gleam command page

### PR DESCRIPTION
**Pull request by michaeljones**
Covers the basics of the command and sub-commands and establishes a format.

For: https://github.com/gleam-lang/gleam/issues/857

### CODE REVIEW

File: documentation.md [hunk 1]
```diff
@@ -27,3 +27,4 @@ layout: page
 - [Gleam's standard library documentation](https://hexdocs.pm/gleam_stdlib/)
 - [The "Awesome Gleam" resource list](https://github.com/gleam-lang/awesome-gleam)
 - [gleam.toml config file reference](/writing-gleam/gleam-toml)
+- [gleam command reference](/writing-gleam/gleam-command)
```
Comment (lpil) [inline documentation.md:30]:
```suggestion
- [Command line reference](/writing-gleam/command-line-reference)
```

File: writing-gleam/gleam-command.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+title: gleam command
```
Comment (lpil) [inline writing-gleam/gleam-command.md:2]:
```suggestion
title: Command line reference￼
```

File: writing-gleam/gleam-command.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+- `--description <description>`: Provide a description which is inserted to the `src/<name>.src.app`
+  file in the new project.
+- `--template <lib|app>`: Indicate whether to generate a project for a library (`lib`) or
+  application (`app`). Defaults to `lib`.
+
+## `build`
+
+`gleam build [project-root]`
+
+Builds the given gleam project. Defaults to the current directory.
```
Comment (lpil) [inline writing-gleam/gleam-command.md:24]: Can we add a note saying this command does not download hex packages for you, and that it is more commonly called indirectly by rebar3? This is often a source of confusion￼

**Review by lpil — COMMENTED** [review]
Looks great! Did you write this manually or is it generated from the command line in some fashion?￼

I've left a couple notes in line, and I've suggested that one of the pages be renamed ￼

**Comment by michaeljones** [comment]
Some lines are copied from the command docs but I wasn't sure there is currently enough in the command docs to auto-generate the page. If you're content with verbose command line docs then I generally have a soft spot for auto-generated documentation.

**Comment by lpil** [comment]
That would be cool. Though given how infrequently this is going to change it might not be worth the time. Could be fun though!￼￼

File: documentation.md (hunk 1 above)
Comment (michaeljones) [reply documentation.md]: Done

File: writing-gleam/gleam-command.md (hunk 2 above)
Comment (michaeljones) [reply writing-gleam/gleam-command.md]: Done

File: writing-gleam/gleam-command.md (hunk 3 above)
Comment (michaeljones) [reply writing-gleam/gleam-command.md]: Done - I hope it reads reasonably.

**Review by lpil — APPROVED** [review]
Thanks!

**Pull request merged**